docs/topics/feed-exports.rst

```diff
@@ -101,12 +101,12 @@ The storages backends supported out of the box are:
 
 -   :ref:`topics-feed-storage-fs`
 -   :ref:`topics-feed-storage-ftp`
--   :ref:`topics-feed-storage-s3` (requires botocore_)
+-   :ref:`topics-feed-storage-s3` (requires boto3_)
 -   :ref:`topics-feed-storage-gcs` (requires `google-cloud-storage`_)
 -   :ref:`topics-feed-storage-stdout`
 
 Some storage backends may be unavailable if the required external libraries are
-not available. For example, the S3 backend is only available if the botocore_
+not available. For example, the S3 backend is only available if at least the botocore_
```

Comment:
Not quite sure how to properly formulate the library requirements in the docs. Strictly speaking, `S3FeedStorage` won't work only there is no botocore, so unless we want to deprecate botocore altogether - we should probably mention the fallback thing in more detail.